day3/slice_time_image.py

```diff
@@ -47,8 +49,32 @@ def slice_time_image(img, slice_times, TR):
         A new copy of the input image with slice-time interpolation applied
     """
     # Get the image data as an array;
+    data = img.get_data()
     # Make a new empty array "interp_data" to hold the interpolated data;
+    interp_data = np.zeros(data.shape)
+    
     # Do the interpolation;
+    time_offset = slice_times/slice_times.shape * TR
```

Comment:
Actually, I was intending that the input array is the thing you are calling `time_offset` here.  I didn't specifically say this - sorry for that - but the clue is in the docstring above "The time in seconds ... relative to the start of the TR ...".